base/applications/mscutils/eventvwr/lang/cs-CZ.rc

```diff
@@ -11,24 +11,24 @@ IDM_EVENTVWR MENU
 BEGIN
     POPUP "&Protokol"
     BEGIN
-        MENUITEM "&Otevřít...", IDM_OPEN_EVENTLOG
-        MENUITEM "Uložit &Protokol jako...", IDM_SAVE_EVENTLOG
+        MENUITEM "&Otevřít...", IDM_OPEN_EVENTLOG//FIXME: accel collision &O
+        MENUITEM "Uložit &Protokol jako...", IDM_SAVE_EVENTLOG//FIXME: accel collision &P
         MENUITEM "&Zavřít", IDM_CLOSE_EVENTLOG
         MENUITEM SEPARATOR
-        MENUITEM "&Odstranit události", IDM_CLEAR_EVENTS
-        MENUITEM "&Přejmenovat\tF2", IDM_RENAME_EVENTLOG
+        MENUITEM "&Odstranit události", IDM_CLEAR_EVENTS//FIXME: accel collision &O
+        MENUITEM "&Přejmenovat\tF2", IDM_RENAME_EVENTLOG//FIXME: accel collision &P
```

Comment:
Please add a space before the `//FIXME` (remark for all of these in the PR)

Reply:
I recently changed to that style without the space intentionally, because I found several translators not deleting the whitespace but just the part beginning with the slashes when acting on the FIXMEs. Reviewing that individually each time to prevent us sticking with that whitespace is not worth the hassle imho. Keep in mind that those comments are a **temporary thing** only, until they are fixed. Ultimately they will be gone, so I think this practice is acceptable.

Reply:
ok, this is reasonable.